python/cudf/cudf/core/resample.py

```diff
@@ -1,4 +1,4 @@
-# SPDX-FileCopyrightText: Copyright (c) 2021 NVIDIA CORPORATION &
+# SPDX-FileCopyrightText: Copyright (c) 2021-2022, NVIDIA CORPORATION &
```

Comment:
Is this copyright correct? Should we rewrite this file's copyright header? @rapidsai/cudf-python-codeowners

Reply:
Looks like this is #9701 (@shwina). Unfortunately that PR has no description to indicate why the copyright was updated.